src/components/Notifications/CompositeAlerts/CustomFilteredAlertModal/CustomFilteredAlertModal.js

```diff
@@ -35,6 +35,10 @@ class CustomFilteredAlertModal extends React.Component {
   EVALUATION_MODE_COMPOSITE = 'COMPOSITE'
   constructor(props) {
     super(props)
+    this.MODAL_TITLES = {
```

Comment:
Is there no Constants file in react-autoql we can add this value to instead of in the constructor?